sdk/servicebus/service-bus/changelog.md

```diff
@@ -1,8 +1,11 @@
 # Coming Soon 1.1.1
 
-- Ensure handlers passed to `registerMessageHandlers` are re-registered when the underlyng receiver encounters
-error and is re-initialized. Fixes [Bug 5541](https://github.com/Azure/azure-sdk-for-js/issues/5541) with
-[PR 5693](https://github.com/Azure/azure-sdk-for-js/pull/5693).
+- When in ReceiveAndDelete mode, skip resolving the promise returned by `receiveMessages` due to a new message not arriving within 1 second of the previous message. This constraint is needed for 
```

Comment:
The peekLock mode may need more investigation and describing it seems unnecessary. We might want to instead describe the bug 5757 as that may be more useful for readers?

Reply:
Without that bit, the previous sentence would have users thinking why the 1 second timer existed in the first place.

The fact that this 1 sec timer is needed for PeekLock mode is well understood as described here and is because we want to avoid earlier messages from getting their locks expired.

Which part needs more investigation?

Reply:
Updated the message to talk about the data loss and removed note on timer